lib/shopify_api/router.ex

```diff
@@ -1,35 +1,34 @@
 defmodule ShopifyAPI.Router do
-  require Logger
   use Plug.Router
+  require Logger
 
+  # alias Absinthe.Plug
   alias GraphQL.Config.Schema
   alias Plug.{Conn, Debugger}
-
-  alias Absinthe.Plug
-
-  alias ShopifyAPI.{App, AppServer, AuthToken, AuthTokenServer, Security}
-
+  alias ShopifyAPI.{App, AuthToken, AuthTokenServer, ConnHelpers}
   alias ShopifyAPI.EventPipe.EventQueue
 
   plug(:match)
   plug(:dispatch)
 
-  if Mix.env() == :dev do
-    use Debugger
-  end
+  #  if Mix.env() == :dev do
```

Comment:
This should probably be removed if it's just going to be commented out?

Reply:
yea
good call
No idea what to do with this still :\

Reply:
It should just be broken out in to its own router or documented